src/components/Chains/ChainSelector/index.tsx

```diff
@@ -0,0 +1,100 @@
+
+import clsx from 'clsx';
+
+import Select, {
+  SelectButton,
+  SelectOptions,
+  SelectOption,
+  SelectBody,
+  SelectCheck,
+  SelectText,
+  SELECT_VARIANTS
+} from 'components/Select';
+
+interface ChainOption {
+  name: string;
+  icon: JSX.Element;
+}
+
+interface Props {
+  chainOptions: Array<ChainOption>;
+  currentChain: ChainOption;
+  onChange: (chain: ChainOption) => void;
+}
+
+const ChainSelector = ({
+  chainOptions,
+  currentChain,
+  onChange
+}: Props): JSX.Element => (
+  <Select
+    variant={SELECT_VARIANTS.formField}
+    key={currentChain.name}
+    value={currentChain}
+    onChange={onChange}>
+    {({ open }) => (
+      <>
+        <SelectBody
+          className={clsx(
+            'w-full'
```

Comment:
@tomjeatt 
I think we do not need this because it's `div` internally which has that behavior by default.